src/pip/_internal/commands/show.py

```diff
@@ -121,6 +121,25 @@ def _get_requiring_packages(current_dist: BaseDistribution) -> Iterator[str]:
 
         metadata = dist.metadata
 
+        project_urls = {}
+        for url in metadata.get_all("Project-URL", []):
+            url_label, url = url.split(",", maxsplit=1)
+            project_urls[url_label.strip()] = url.strip()
```

Comment:
This parsing code does not need to be executed if `Home-page` is present. Let’s move it into the conditional block. Also I don’t think the dict really needs to be built.

Reply:
Fair enough :+1: